utils/query_engine/subquestion_engine.py

```
@@ -0,0 +1,81 @@
+from typing import Sequence
+from typing import List, Optional, Sequence, cast
```

Ruff (0.8.0)

Redefinition of unused `Sequence` from line 1

Remove definition: `Sequence`

(F811)